src/actions/getDeviceDetails.ts

```diff
@@ -0,0 +1,34 @@
+import { withActionOutput } from '../utils';
+
+import WFDeviceDetail from '../interfaces/WF/WFDeviceDetail';
+import WFSerialization from '../interfaces/WF/WFSerialization';
+import WFWorkflowAction from '../interfaces/WF/WFWorkflowAction';
+
+/**
+ * Get Device Details Action. Returns a particular property of the device.
```

Comment:
Could this be changed to _Get Device Details Action. Gets information about the current device._ please? Just to match the description in the Shortcuts app.